llvm/test/CodeGen/AMDGPU/insert_extract_element.ll

```diff
@@ -0,0 +1,218 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: llc -mtriple=amdgcn--amdhsa -mcpu=gfx90a -verify-machineinstrs < %s | FileCheck -check-prefixes=GFX90a %s
+; RUN: llc -mtriple=amdgcn--amdhsa -mcpu=gfx940 -verify-machineinstrs < %s | FileCheck -check-prefixes=GFX940 %s
+; RUN: llc -mtriple=amdgcn--amdhsa -mcpu=gfx1030 -verify-machineinstrs < %s | FileCheck -check-prefixes=GFX1030 %s
+; RUN: llc -mtriple=amdgcn--amdhsa -mcpu=gfx1100 -verify-machineinstrs < %s | FileCheck -check-prefixes=GFX1100 %s
+
```

Comment:
Test name is very general, and we already have tests for insertelement/extractelement. What is this testing that's new? Can it be merged into an existing test?

Reply:
insert_vector_elt.ll looks like a good candidate. Will include in there instead.

Reply:
It's still not really a test of this. If it's going to be standalone should name it something relevant to the problem you were looking for